misc/deployments/test4.gno.land/config.toml

```diff
@@ -0,0 +1,247 @@
+
+# TCP or UNIX socket address for Tendermint to listen on for
+# connections from an external PrivValidator process
+priv_validator_laddr = ""
+
+# Path to the JSON file containing the last sign state of a validator
+priv_validator_state_file = "secrets/priv_validator_state.json"
+
+# TCP or UNIX socket address for the profiling server to listen on
+prof_laddr = ""
+
+# TCP or UNIX socket address of the ABCI application, 
+# or the name of an ABCI application compiled in with the Tendermint binary
+proxy_app = "tcp://127.0.0.1:26658"
+
+##### consensus configuration options #####
+[consensus]
+
+  # EmptyBlocks mode and possible interval between empty blocks
+  create_empty_blocks = true
+  create_empty_blocks_interval = "0s"
+  home = ""
+
+  # Reactor sleep duration parameters
+  peer_gossip_sleep_duration = "100ms"
+  peer_query_maj23_sleep_duration = "2s"
+
+  # Make progress as soon as we have all the precommits (as if TimeoutCommit = 0)
+  skip_timeout_commit = false
+  timeout_commit = "2s" # Do NOT change me, leave me at 2s!
+  timeout_precommit = "1s"
+  timeout_precommit_delta = "500ms"
+  timeout_prevote = "1s"
+  timeout_prevote_delta = "500ms"
+  timeout_propose = "3s"
+  timeout_propose_delta = "500ms"
+  wal_file = "wal/cs.wal/wal"
+
+##### mempool configuration options #####
+[mempool]
+  broadcast = true
+
+  # Size of the cache (used to filter transactions we saw earlier) in transactions
+  cache_size = 10000
+  home = ""
+
+  # Limit the total size of all txs in the mempool.
+  # This only accounts for raw transactions (e.g. given 1MB transactions and
+  # max_txs_bytes=5MB, mempool will only accept 5 transactions).
+  max_pending_txs_bytes = 1073741824
+  recheck = true
+
+  # Maximum number of transactions in the mempool
+  size = 10000 # Advised value is 10000
+  wal_dir = ""
+
+##### peer to peer configuration options #####
+[p2p]
+
+  # Toggle to disable guard against peers connecting from the same ip.
+  allow_duplicate_ip = false
+  dial_timeout = "3s"
+
+  # Address to advertise to peers for them to dial
+  # If empty, will use the same port as the laddr,
+  # and will introspect on the listener or use UPnP
+  # to figure out the address.
+  external_address = ""
+
+  # Time to wait before flushing messages out on the connection
+  flush_throttle_timeout = "100ms"
+
+  # Peer connection configuration.
+  handshake_timeout = "20s"
+  home = ""
+
+  # Address to listen for incoming connections
+  laddr = "tcp://0.0.0.0:26656" # Change me!
+
+  # Maximum number of inbound peers
+  max_num_inbound_peers = 40
+
+  # Maximum number of outbound peers to connect to, excluding persistent peers
+  max_num_outbound_peers = 40 # Advised value is 40
+
+  # Maximum size of a message packet payload, in bytes
+  max_packet_msg_payload_size = 1024
+
+  # Comma separated list of nodes to keep persistent connections to
+  persistent_peers = ""
+
+  # Set true to enable the peer-exchange reactor
+  pex = true
```

Comment:
FYI, for `validator node` this option should be `false`

Reply:
Updated:

[5aeedac](https://github.com/gnolang/gno/pull/2508/commits/5aeedacf3b6810753464c26f2436121848f6fa77)